backend/src/main/java/gov/cdc/usds/simplereport/api/pxp/PatientExperienceController.java

```diff
@@ -91,7 +91,7 @@ public PxpVerifyResponse getPatientLinkVerify(
     OrderStatus os = pl.getTestOrder().getOrderStatus();
```

Comment:
Couldn't much of this method be replaced with lookups in the current patient context?
```
    TestOrder to = _currentPatientContext.getTestOrder();
    Person p = to.getPatient();
    TestEvent te = to.getTestEvent();
```

Reply:
Yes. I've got a follow-up PR to this that cleans up some other things and fully removes `TestEventService` since it's orphaned as a result of this change.

I'll make this improvement there.

Reply:
Actually, the failing e2e test revealed that fixing this bug isn't quite so simple. I'm going to make this improvement now, along w/ the broader bug fix. Will ping you for re-review